alpha/datarepomonitoring/kube-prometheus-stack.yaml

```diff
@@ -72,7 +72,7 @@ grafana:
   dashboards:
     default:
       jvm-dashboard:
-        url: https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/add-grafana-dashboard/perf/datarepomonitoring/dashboards/prometheus-metrics.json
+        url: https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dashboards/jvm-dashboard.json
```

Comment:
I've curious why alpha looks different than staging and prod?

Reply:
Oh the diff here is different because Mark already included the alpha and perf changes in his umbrella chart PR.

Reply:
gotcha, thanks!